ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -498,14 +499,14 @@ public void testCreateStreamWithTopic() {
     final String
         queryStr =
         "CREATE STREAM orders (ordertime bigint, orderid varchar, itemid varchar, orderunits "
-        + "double) WITH (registered_topic = 'orders_topic' , key='ordertime');";
+        + "double) WITH (registered_topic = 'orders_topic' , key='ordertime', kafka_topic='foo', value_format='json');";
```

Comment:
I thought `registered_topic` was no longer supported...

Reply:
That's what I thought as well. I just kept this in tact - if we want to remove it from syntax that's for another PR